Add `options` prop to <SelectField> (#1342)
aws-amplify/amplify-ui: docs/src/pages/components/selectfield/react.mdx

  </ExampleCode>
</Example>

### Options

> To create a simple SelectField in which each `<option>` has matching `value`, `label` and `children`, pass a `string[]` to the `options` prop.

Reviewer: Maybe leave the type information out of the docs, in case it changes and doesn't stay in sync. 
```suggestion
Create a simple `SelectField` by passing a list to the `options` prop.
```
